src/ralph/backends/data/base.py

```diff
@@ -294,6 +295,7 @@ def read(  # noqa: PLR0913
             ignore_errors (bool): If `True`, encoding errors during the read operation
                 will be ignored and logged.
                 If `False` (default), a `BackendException` is raised on any error.
+            max_statements (int): The maximum number of statements to yield.
```

Comment:
```suggestion
            max_statements (int): The maximum number of statements to yield. If `None` (default), there is no maximum.
```

Idem for similar docstrings.

Reply:
Good Idea) Thanks) Added it)